go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/consensys/gnark
 
-go 1.19
+go 1.21
```

Comment:
need to keep to go 1.20 I think -- probably my bad wanted to use the (cough cough) awesome builtin "max" and "min" function